maintenance/fuel-slaves.sh

```diff
@@ -50,7 +50,7 @@ do
 	 --arch=x86_64 \
 	 $virt_disks_params \
 	 $virt_net_params \
-	 --boot network \
+	 --boot network,hd \
```

Comment:
The maintenance scripts create new nodes, there's nothing to boot from besides network at that point.